test/e2e/flowexporter_test.go

```diff
@@ -51,7 +53,25 @@ func TestFlowExporter(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Error when getting the perftest server Pod's IP: %v", err)
 	}
-	stdout, _, err := data.runCommandFromPod(testNamespace, "perftest-a", "perftool", []string{"bash", "-c", fmt.Sprintf("iperf3 -c %s|grep sender|awk '{print $7,$8}'", podBIP)})
+
+	if clusterInfo.podV4NetworkCIDR != "" {
+		if podAIP.ipv4 == nil || podBIP.ipv4 == nil {
+			t.Fatalf("No IPv4 address is assigned for Pod")
+		}
```

Comment:
why this check instead of using `skipIfNotIPv4Cluster`?

Reply:
Make sense, change to use `skipIfNotIPv4Cluster`